tasks/push-rpm-manifests-to-pyxis/README.md

```diff
@@ -11,6 +11,9 @@ Tekton task that extracts all rpms from the sboms and pushes them to Pyxis as an
 | server | The server type to use. Options are 'production','production-internal,'stage-internal' and 'stage'. | Yes | production |
 | concurrentLimit | The maximum number of images to be processed at once | Yes | 4 |
 
+## Changes in 0.3.0
+* updated the base image used in this task
```

Comment:
does it make sense to add some brief details that this was done to fix the bug XYZ?

Reply:
I just copied the description that was used last time something like this was done. The link to the PR is in the commit itself.